wrapper/src/test/java/integration/refactored/container/tests/ReadWriteSplittingTests.java

```diff
@@ -101,32 +101,32 @@ protected String queryInstanceId(Connection conn) throws SQLException {
   protected String getUrl() {
     return DriverHelper.getWrapperDriverProtocol()
         + TestEnvironment.getCurrent()
-            .getInfo()
-            .getDatabaseInfo()
-            .getInstances()
-            .get(0)
-            .getEndpoint()
+        .getInfo()
+        .getDatabaseInfo()
+        .getInstances()
+        .get(0)
+        .getEndpoint()
```

Comment:
Can we please try to avoid code style changes that aren't failing checkstyle in future PRs? This introduce quite a bit of noise to the PR